main.css

```
@@ -4,6 +4,36 @@
     box-sizing: border-box;
 }
 
+/* Mobile Styles */
+@media only screen and (max-width: 400px) {
+    body {
+        font-family: Arial;
+        color: black;
+        background-color:#FFCB77;
+        font-weight: normal;
+        font-size: 14px;
+    }
+  }
+  
+  /* Tablet Styles */
+  @media only screen and (min-width: 401px) and (max-width: 960px) {
+    body {
+      background-color: #F5CF8E; /* Yellow */
+    }
+  }
+  
+  /* Desktop Styles */
+  @media only screen and (min-width: 961px) {
+    body {
+        font-family: Arial;
+        color: white;
+        background-color:#5E5C6C;
```

💭 Give your declarations some space. 
🙃 Also, it would be nice to see a comment next to the color to tell us what color it actually is like you did on line 21. 
```suggestion
        background-color: #5E5C6C;
```

---

These are for @thetechtakeover to fix in issue #8